templating/values_test.go

```diff
@@ -139,7 +137,6 @@ func TestOverrideValuesWithBuildInfo(t *testing.T) {
 		// Base properties
 		{"{{.Build.ID}}", expectedID},
 		{"{{.Build.Commit}}", expectedCommit},
-		{"{{ .Build.Tag }}", expectedTag},
 		{"{{ .Build.Repository}}", expectedRepo},
```

Comment:
>  [](start = 6, length = 1)

extra space

Reply:
@northtyphoon this one's on purpose, to make sure we can render it correctly with arbitrary white spaces on the side